pkg/mobile/mobileClient_test.go

```diff
@@ -0,0 +1,62 @@
+package mobile
+
+import (
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/operator-framework/operator-sdk/pkg/sdk"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+func TestMobileHandler(t *testing.T) {
+	t.Run("test mobile handler", func(t *testing.T) {
+		handler = &MobileHandler{}
+		watcher1 := MobileWatcher{events: make(chan watch.Event)}
+		watcher2 := MobileWatcher{events: make(chan watch.Event)}
+		handler.watchers = append(handler.watchers, watcher1)
+		handler.watchers = append(handler.watchers, watcher2)
+		go func() {
+			for index := 0; index < 2; index++ {
+				handler.Handle(nil, sdk.Event{Object: nil, Deleted: false})
+			}
```

Comment:
why call the handler twice here?

Reply:
I just wanted to verify, that
- when there are watchers registered, they all receive notification
- and when watcher unregisters [1], it will not receive more notifications, but the rest of the watchers will do

[1] https://github.com/aerogear/mobile-developer-console/pull/71/files#diff-b5c34eb680cd64a7334e27350c53c584R36

Reply:
right, now I get it. Might worth adding some comments to explain it. Thanks.